nomos-da/kzgrs-backend/src/common/mod.rs

```diff
@@ -165,6 +171,10 @@ pub fn commitment_to_bytes(commitment: &Commitment) -> Vec<u8> {
     buff.into_inner()
 }
 
-pub struct Attestation {
-    pub signature: Signature,
+fn hash(item: impl AsRef<[u8]>) -> [u8; 32] {
```

Comment:
Lets call it by a proper name `blake2b_hash`.
Also, why using the `Blake2bVar` version for the standard `32bytes` size?

Reply:
Removed this method and used Blake2b (default 32) in the attestation directly.